flink-doris-connector/src/test/java/org/apache/doris/flink/utils/DateToStringConverter.java

```diff
@@ -43,23 +43,27 @@ public class DateToStringConverter implements CustomConverter<SchemaBuilder, Rel
     private DateTimeFormatter timestampFormatter = DateTimeFormatter.ISO_DATE_TIME;
     private ZoneId timestampZoneId = ZoneId.systemDefault();
 
-    public static Properties DEFAULT_PROPS = new Properties();
+    public static Properties defaultProps = new Properties();
```

Comment:
Does it have to be static final to be capitalized?

Reply:
Yes, changed